proto/one/historian/ingest/ingest_client.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+import "enum_log_level.proto";
+import "file_configuration.proto";
+
+option csharp_namespace ="ONE.Models.CSharp";
+
+message IngestClient
+{
+     // Client name
+     string name = 1;
+
+     // Client SubType Guid
+     string subTypeId = 2;
+
+     // Whether the client is enabled
+     bool enabled = 3;
+
+     // Log Level for the Client
+     EnumLogLevel logLevel = 4;
+
+     // Frequency in seconds the client should check for configuration updates
+     uint32 configCheckFrequencySeconds = 5;
+
+     // Frequency in seconds the client should run data source drivers
+     uint32 cycleFrequencySeconds = 6;
```

Comment:
What about `string` here instead of a number, we could then store the format we want instead of converting the duration to a total number of seconds. Could be handy if we switch over to using an ISO duration like `PT5H`, though that isn't natively supported by .NET so unlikely we do this anytime soon.

A second is probably the smallest granularity we need for this so it is probably fine.

Reply:
Seems reasonable, we can always adjust later on if this proves problematic. 

Is there a reason we cant use the [duration](https://github.com/protocolbuffers/protobuf/blob/main/src/google/protobuf/duration.proto) datatype? (I'm pretty ignorant to all this stuff).

Reply:
using int for duration seems to be the existing pattern; https://github.com/AquaticInformatics/ONE.Interfaces.ProtocolBuffers/blob/main/proto/one/common/core/timewindow.proto
although i had, out of curiosity, tried [duration](https://github.com/protocolbuffers/protobuf/blob/main/src/google/protobuf/duration.proto)   but found that i couldn't get the json request to serialize correctly.  not sure if i was missing something there

Reply:
Fair enough, seems reasonable to leave it as-is then.